test/unit/mongo/MongoTripodViewsTest.php

```diff
@@ -342,13 +379,39 @@ public function testGetViewForResourcesInvokesViewGenerationForMissingResources(
         $context = "http://someContext";
 
         $mockDb = $this->getMock("MongoDB", array("selectCollection"),array(new MongoClient(),"test"));
-        $mockColl = $this->getMock("MongoCollection", array("findOne"),array($mockDb,$this->tripod->collection->getName()));
+        $mockColl = $this->getMock("MongoCollection", array("findOne"),array($mockDb,$this->tripod->getPodName()));
+        $mockViewColl = $this->getMock("MongoCollection", array("findOne"),array($mockDb,VIEWS_COLLECTION));
 
-        $mockDb->expects($this->once())->method("selectCollection")->will($this->returnValue($mockColl));
+        $mockDb->expects($this->any())->method("selectCollection")->will($this->returnValue($mockColl));
```

Comment:
I mean, I agree we don't want to use ``` any() ``` when we actually want to be sure something happens.  I just don't think that's the expectation here.